docs/reference/src/main/asciidoc/events.asciidoc

```diff
@@ -93,25 +100,66 @@ injection:
 @Inject @Any Event<Document> documentEvent;
 -------------------------------------------
 
-A producer raises events by calling the `fire()` method of the `Event`
+==== Synchronous event producers
+A producer raises synchronous events by calling the `fire()` method of the `Event`
 interface, passing the event object:
 
 [source.JAVA, java]
 -----------------------------
 documentEvent.fire(document);
 -----------------------------
 
-This particular event will be delivered to every observer method that:
+This particular event will be delivered only to synchronous observer method that:
```

Comment:
I think the position of `only` in the sentence isn't correct. How about:
`This particular event will only be delivered to synchronous observer method that:`
If you decide to change it, please do the same in the async part of this chapter.